sdk/unity/xr_provider/input.cc

```diff
@@ -128,7 +128,8 @@ class CardboardInputProvider {
   UnitySubsystemErrorCode Tick() {
     std::array<float, 4> out_orientation;
     std::array<float, 3> out_position;
-    cardboard_api_->GetHeadTrackerPose(out_position.data(),
+    cardboard_api_->GetHeadTrackerPose(kCardboardViewportOrientationLandscapeLeft,
```

Comment:
by now, instead of passing the orientation as a parameter, let's just set it within the method.